e2e/tests/components/toolbar-search/toolbar-search.e2e-spec.ts

```diff
@@ -31,6 +31,14 @@ describe('Toolbar Search', () => {
 
     });
 
+    it('should display the name of the selected number', async () => {
+
+        await page.clickOnDropdown();
+        expect(await page.confirmDropdownIsExpanded()).toBeFalsy();
```

Comment:
Should this not be true? The dropdown looks open in the screenshot? Or maybe I'm misunderstanding the purpose of this test